woocommerce/woocommerce-ios
# REST API: Migrate JITM endpoints

Closes: #8661 


## Description

This PR continues the work for the REST API migration by enabling REST API for JITM endpoints.

## Testing instructions

Pre-requisite: Make sure to have a self-hosted store and follow pdfdoF-1uc-p2#comment-2581 for details of setting up your store to be eligible for the test JITM.
- Enable the feature flag `applicationPasswordAuthenticationForSiteCredentialLogin` and build the app.
- On the prologue screen, select Enter your site address and proceed with the address of your test store.
- Log in with your site credentials. After the login succeeds, you should be navigated to the home screen.
- On the home screen, notice the JITM banner is displayed. Xcode console should log `🔵 Tracked jitm_fetch_success` and `🔵 Tracked jitm_displayed`.
- Tap the dismiss button on the banner, the banner should disappear. Xcode console should log `🔵 Tracked jitm_dismissed` and `🔵 Tracked jitm_dismiss_success`

## Screenshots

<img src="https://user-images.githubusercontent.com/5533851/212818640-1d4144e6-6186-4a67-a109-48383391cb9f.png" width=320 />


---
